tests/data/other_cert.pem

```diff
@@ -0,0 +1,33 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
There should be a note in the tests file describing how to regenerate test certs.